internal/store/atlas/teams.go

```diff
@@ -15,7 +15,7 @@
 package atlas
 
 import (
-	atlasv2 "go.mongodb.org/atlas-sdk/admin"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20230201001/admin"
```

Comment:
Every import has now major version. 
🥇 We can update by replacing imports across the board
☄️  for compatibility we can leave old imports when we see breaking changes for end users
🥈  we can use multiple versions of SDK in the CLI and update on demand ensuring consistency, backwards compatibility and also autodiscovery of new changes